src/main/java/org/spongepowered/api/world/extent/Extent.java

```diff
@@ -25,10 +25,11 @@
 
 package org.spongepowered.api.world.extent;
 
-import com.flowpowered.math.vector.Vector3i;
 import org.spongepowered.api.block.BlockLoc;
 import org.spongepowered.api.world.weather.WeatherUniverse;
 
+import com.flowpowered.math.vector.Vector3i;
```

Comment:
Are you sure? You need to import import order, cleanup and formatter separately - if that was done correctly it shouldn't screw up import anymore, at least not after you run Organize Imports for the file